hotfixes/wc-calypso-bridge-mailchimp-no-redirect.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * add_settings_error is only available in UI contecext
```

Comment:
I think this brought over the wrong file: https://github.com/woocommerce/wc-api-dev/blob/master/hotfixes/wc-api-dev-mailchimp-no-redirect.php